nextflow.config

```diff
@@ -1,85 +1,142 @@
-// =================================================================
-// nextflow.config is the main configuration script for a nextflow pipeline
-// Should contain the following sections:
-	// A manifest for defining workflow metadata
-	// Mandated minimal version of Nextflow required to run pipeline
-	// Default workflow parameter definitions
-	// Shell behaviour settings for the workflow
-	// Execution reports
-	// Configuration profiles
-	// Default resource definitions for processes
-
-// Examples are included for each section. Remove them and replace
-// with project-specific code. For more information see:
-// https://www.nextflow.io/docs/latest/index.html.
-//
-// ===================================================================
-
-// Define some metadata for publishing pipeline on GitHub
-// See https://www.nextflow.io/docs/latest/config.html#scope-manifest
+// Workflow details
 manifest {
-	author = 'Georgie Samaha'
-	name = 'Nextflow_DSL2_template-nf'
-	description = 'Template for creating Nextflow workflows'
-	homePage = 'https://github.com/Sydney-Informatics-Hub/Nextflow_DSL2_template'
-}
+    
+  author = ' Tracy Chew, Cali Willet, Nandan Deshpande, Sydney Informatics Hub, University of Sydney'
+  name = 'SomaticShortV-nf'
+  description = 'GATK somatic short variant calling pipeline'
+  homePage = 'https://github.com/Sydney-Informatics-Hub/Somatic-shortV-nf'
+    }
 
 // Mandate a minimum version of nextflow required to run the pipeline
-// See https://www.nextflow.io/docs/latest/config.html#nextflow-version
-nextflowVersion = '!>=20.07.1'
+nextflowVersion     = '!>=20.07.1'
 
-// Set default workflow parameters
-// See https://www.nextflow.io/docs/latest/config.html#scope-params 
+// Set default parameters
 params {
-	help	= false
-	outDir	= 'results'
-	input	= false
+  version                       = '1.0'
+  help                          = false
+  input                         = null
+  ref                           = false
+  intervals                     = false
+  outDir		                    = './results'
+  ponvcf                        = null
+  small_exac_common             = ''
+  intervalList_path             = ''
+  intervalList          = ['0000','0001','0002','0003','0004','0005','0006','0007','0008','0009','0010','0011','0012']
+
+  
+  
+  
+  gadi_account       	= false
+  setonix_account     = false
+  whoami            	= false
 }
 
+
+// Preset use of containers with Singularity
+singularity {
+    enabled		    = false
+    autoMounts		= true
+    }
+
+// Job profiles
+profiles {
+  local		{ includeConfig "config/local.config" }
+  gadi		{ includeConfig "config/gadi.config" }
+  }
+
 // Fail a task if any command returns non-zero exit code
-// See https://www.nextflow.io/docs/latest/process.html#script 
 shell = ['/bin/bash', '-euo', 'pipefail']
 
+// Resources for each process
+// default run resource parameters
+process {
+
```

Comment:
Can you add container here instead of repeating under each withName

Reply:
Done